webpack.config.js

```diff
@@ -36,16 +39,21 @@ function getLoadersConfig(env) {
 }
 
 function getPluginsConfig(env) {
+  if(!config.availableLanguages) {
+    throw "availableLanguages needs to be configured in the config file";
+  }
+  var languageExpression = new RegExp("^./(" + config.availableLanguages.join('|') + ")$");
```

Comment:
We could loop through the config files, but should we have to depend on configuration at all during build time? Maybe it's hard to avoid.